assets/core/css/custom-pdf.css

```
@@ -3,4 +3,621 @@
    Custom Stylesheet
 
    All changes made in this file will override the default styles in PDFs.
-   ========================================================================== */
\ No newline at end of file
+   ============================================================================*/
```

Stylelint (17.7.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)